wordle-clone/src/game.vue

```diff
@@ -0,0 +1,86 @@
+<script setup lang="ts">
```

Comment:
This should be under your views folder and the file name should be PascalCase `Game.vue`